executor/src/witgen/bus_accumulator/mod.rs

```diff
@@ -131,17 +159,28 @@ impl<'a, T: FieldElement> BusAccumulatorGenerator<'a, T> {
                 false => current_acc + folded.inverse() * multiplicity,
             };
 
-            folded1[i] = folded.0;
-            folded2[i] = folded.1;
-            acc1[i] = new_acc.0;
-            acc2[i] = new_acc.1;
+            folded_list[i] = folded;
+            acc_list[i] = new_acc;
+        }
+
+        // Transpose from row-major to column-major & flatten.
```

Comment:
This is the only real change to the GL witgen with this PR. I checked that both on `main` and on this branch, second-stage witgen takes around 8.5% of proving time on the Keccak example with `--linker-mode bus`.